src/js/main.js

```diff
@@ -27,23 +27,18 @@ import { sync } from 'vuex-router-sync'
 import store from './store/index.js'
 import router from './router.js'
 import ClickOutside from 'v-click-outside'
-import { getRequestToken, getCurrentUser } from '@nextcloud/auth'
+import { getCurrentUser } from '@nextcloud/auth'
 import { translate, translatePlural } from '@nextcloud/l10n'
-import { generateFilePath } from '@nextcloud/router'
 import { Tooltip } from '@nextcloud/vue'
 
 import UserItem from './components/User/UserItem.vue'
 
-/* eslint-disable-next-line camelcase, no-undef */
-__webpack_nonce__ = btoa(getRequestToken())
-/* eslint-disable-next-line camelcase, no-undef */
-__webpack_public_path__ = generateFilePath('polls', '', 'js/')
-
 sync(store, router)
 
-Vue.config.debug = process.env.NODE_ENV !== 'production'
-Vue.config.devTools = process.env.NODE_ENV !== 'production'
-
+Vue.config.debug = true
+Vue.config.devTools = true
+console.debug('import.meta.env.DEV', import.meta.env.DEV, 'import.meta.env.PROD', import.meta.env.PROD, 'import.meta.env.MODE', import.meta.env.MODE, 'process.env.VUE_APP_DEBUG', process.env.VUE_APP)
+console.debug('Vue.config.devTools', Vue.config.devTools, 'Vue.config.debug', Vue.config.debug)
```

Comment:
Some notes:
- `VUE_APP_` - is a prefix from Vue CLI. It is not available neither in Vite nor in any custom Webpack config other than Vue CLI service
- `MODE` and `DEV/PROD` have very different meaning:
  - `MODE` is about Vite building mode. By default, it is `production` for `vite build` and `development` for `vite`. But `--mode=X` overrides this. In other words, it is about how it is being bundled (using minification, debug and ect.). So this is what you want to use here
  - `DEV` and `PROD` only reflects `NODE_ENV` variable, with is always `development` for dev server (not used here) and `production` for building app `vite build` regardless of `--mode`.
  - See also: https://vitejs.dev/guide/env-and-mode.html#node-env-and-modes

Reply:
Probably, we should manually set `NODE_ENV` to `development` when `--mode=development` in our vite config

Reply:
Thanks for the explanation.